### ISSUE TITLE
Bad material style support

### ISSUE THREAD

**Issue by ivanshilo** [juristr/egghead-ngx-formly-course#26]
Please be aware that styling is not exactly matches material style. And it is pretty annoying. The issue is not solved yet and seems that nobody cares. This makes this widget kind of useless for smooth integration. https://github.com/ng-select/ng-select/issues/1868

**Comment by VLEFF** [comment]
Same problem here, disappointed =/

**Comment by juristr** [comment]
yeah I know. Cannot do anything though since it is an external lib. Maybe someone could step in and provide a fix?
